definitions/infra-awslex/golden_metrics.yml

```diff
@@ -0,0 +1,36 @@
+requests:
+  title: Total Request Count
+  unit: COUNT
+  queries:
+    aws:
+      select: sum(aws.lex.RuntimeRequestCount)
+      from: Metric
+      eventId: entity.guid
+      eventName: entity.name
+RuntimeSystemErrors:
```

Comment:
Use first lowercase for first letter of metric names

Reply:
addressed